uint32 sequence_id
int8 exec_level
string instruction
int8 feedback_level
string feedback
uint32 feedback_id
string[] string_values
float32[] float_values

